test-data/unit/check-typevar-defaults.test

```diff
@@ -315,7 +318,7 @@ def func_c3(
 class ClassC4(Generic[T1, Unpack[Ts1], T3]): ...
 
 def func_c4(
-    a: ClassC4,
+    a: ClassC4,  # E: Missing type parameters for generic type "ClassC4[()]"
```

Comment:
Why the `[()]` part?

Reply:
Adjusted the type formatting to better handle this case. It was just checking if the instance has a `TypeVarTuple` and assumed that it's the only TypeVar -> thus printing `()` for an empty tuple.

Reply:
Thanks!